test/functional/p2p_segwit.py

```diff
@@ -92,7 +92,7 @@
 
 MAX_SIGOP_COST = 80000
 
-SEGWIT_HEIGHT = 120
+SEGWIT_HEIGHT = 260
```

Comment:
I don't think this is correct? Can we disable this test?

Reply:
Again can, but I'd suggest it's the next PR

Reply:
Ah, so what this does is override the default from `chainparams.cpp`. We could remove the test if we're going to rip the SegWit code out entirely, but in this specific case even if the default chainparams were never enable SegWit, the test should still override here.